torchvision/io/image.py

```diff
@@ -132,17 +137,23 @@ def write_png(input: torch.Tensor, filename: str, compression_level: int = 6):
     write_file(filename, output)
 
 
-def decode_jpeg(input: torch.Tensor) -> torch.Tensor:
+def decode_jpeg(input: torch.Tensor, components: int = 0) -> torch.Tensor:
```

Comment:
Here is the channels/components discussion again.

Reply:
I would really prefer if at least the python API has the same argument names.

Reply:
I agree, I think I should just replace everything with channels for the shake of simplicity in semantics.

Reply:
done